chapters/classes.tex

```diff
@@ -217,17 +217,31 @@ \subsubsection{Prefix Rules}\label{prefix-rules}
 
 A \firstuse{prefix} is property of an element of a class definition which can be present or not be present, e.g., \lstinline!final!, \lstinline!public!, \lstinline!flow!.
 
-Variables declared with the \lstinline!flow! or the \lstinline!stream! type prefix shall be a subtype of \lstinline!Real!.
-
-Type prefixes (that is, \lstinline!flow!, \lstinline!stream!, \lstinline!discrete!, \lstinline!parameter!, \lstinline!constant!, \lstinline!input!, \lstinline!output!) shall only be applied for type,
-record and connector components -- see also record specialized class, \cref{specialized-classes}.
+Type prefixes (that is, \lstinline!flow!, \lstinline!stream!, \lstinline!discrete!, \lstinline!parameter!, \lstinline!constant!, \lstinline!input!, \lstinline!output!) shall only be applied for type, record, operator record, and connector components -- see also record specialized class, \cref{specialized-classes}.
+This is further restricted below; some of these combinations of type prefixes and specialized classes are not legal.
 
 An exception is \lstinline!input! for components whose type is of the special class function type (these can only be used for function formal parameters and has special semantics, see \cref{functional-input-arguments-to-functions}), and the \lstinline!input! prefix is not applied to the elements of the component and is allowed even if the elements have \lstinline!input! or \lstinline!output! prefix.
 
 In addition, instances of classes extending from \lstinline!ExternalObject! may have type prefixes \lstinline!parameter! and \lstinline!constant!, and in functions also type prefixes
 \lstinline!input! and \lstinline!output!, see \cref{external-objects}.
```

Comment:
Can we fix the line break here?

Reply:
I was unsure what you wanted - I added a line-break above. If that wasn't correct, please explain.

Reply:
Sorry, that was not clear. I meant the part on line 227 is actually the same sentence that started on line 226. According to the guidelines, we should have one sentence on one line. And this sentence seems to have not been updated. Unless there is some reason we kept it on two lines? That's not really part of this PR but while we are changing this chapter and while I noticed it, I thought it might be ok to change it here instead of opening a separate PR.

Reply:
Ok, I undid the previous one and merged that line.